libecole/include/ecole/reward/dualbound.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "ecole/reward/abstract.hpp"
+#include "ecole/scip/type.hpp"
+
+#define infinity   1e+20
```

Comment:
I think we should rather use SCIP's infinity, which should be available somewhere from SCIP